ghost/i18n/locales/tr/portal.json

```diff
@@ -95,8 +95,8 @@
     "In the event a permanent failure is received when attempting to send a newsletter, emails will be disabled on the account.": "Haber bülteni göndermeye çalışırken kalıcı bir hata alınması durumunda, hesapta e-postalar devre dışı bırakılır.",
     "In your email client add {{senderEmail}} to your contacts list. This signals to your mail provider that emails sent from this address should be trusted.": "E-posta istemcinizde {{senderEmail}} adresini kişi listenize ekleyin. Bu, posta sağlayıcınıza bu adresten gönderilen e-postaların güvenilir olması gerektiğini bildirir.",
     "Invalid email address": "Geçersiz e-posta adresi",
-    "Jamie Larson": "Ahmet Yılmaz",
-    "jamie@example.com": "ahmet@example.com",
+    "Jamie Larson": "Ad Soyad",
+    "jamie@example.com": "kullanıcıadı@example.com",
```

Comment:
Similarly, this update moves away from the original intent as I believe it says "username@example.com"

Reply:
Usually, the username from the email is displayed. Therefore, it is specified to enter their own email username. However, in the current translation, someone has written their own name, and it doesn't look appropriate on the site. Based on this, I wanted to clarify the correct approach.